### DISCUSSION TITLE
[Docs] How to drop support for IE11

### DISCUSSION THREAD

**Discussion by matamatanot** [vercel/next.js#17586] [Ideas]
Next.js supports IE11 by default.
However, there is little need to support IE11 in the future.
Due to security concerns, there are some services that dare to forbid the use of IE11.

How can you exclude IE11 support in Next.js?
Is it worth adding to the documentation?

**Comment by leerob** [comment]
You can customize which browsers you want to support by overriding your Babel config. Once IE11 is dead, it will likely be removed as a default in Next.

https://nextjs.org/docs/advanced-features/customizing-babel-config

**Comment by matamatanot** [comment]
@leerob

> Once IE11 is dead, it will likely be removed as a default in Next.

That's interesting.
It would be nice to be able to disable IE11 early on in next.config.js.

The actual .babelrc was commented on #17822.

**Comment by GrossDesignCo** [comment]
+1 to this ability at least being documented somewhere. We have a number of internal modules that explicitly ship es6, and have dropped IE11 and Pre-chromium edge support entirely. It would be great to simplify our configs & not have to transpile those internal modules.

**Comment by nathggns** [comment]
Would be good to see how to remove IE11 support (particularly to avoid async/await being transpiled to generators) while using SWC, not babel. Any tips?
